packages/expo-cli/src/commands/credentials.ts

```diff
@@ -20,8 +25,23 @@ export default function (program: CommanderStatic) {
     .description('Manage your credentials')
     .helpGroup('credentials')
     .option('-p --platform <platform>', 'Platform: [android|ios]', /^(android|ios)$/i)
+    .option('--config [file]', 'Specify a path to app.json or app.config.js')
     .asyncAction(async (options: Options) => {
       const projectDir = process.cwd();
+      // @ts-ignore: This guards against someone passing --config without a path.
+      if (options.config === true) {
+        log('Please specify your custom config path:');
+        log(
+          log.chalk.green(`  expo credentials:manager --config ${log.chalk.cyan(`<app-config>`)}`)
+        );
+      }
+      if (options.config) {
+        const configPath = path.resolve(projectDir, options.config);
+        if (!(await fs.pathExists(configPath))) {
+          throw new Error(`File ${configPath} does not exist`);
+        }
+        ConfigUtils.setCustomConfigPath(projectDir, configPath);
```

Comment:
iirc this particular method is somewhat fragile and shouldn't be used in more than one place.